arch/sim/src/sim/up_setjmp32.S

```diff
@@ -46,8 +46,12 @@
 #ifdef __CYGWIN__
 # define SYMBOL(s) _##s
 #else
+#ifdef __ELF__
```

Comment:
The logic is wrong, should it be #ifndef?

Reply:
oops, you are right.

Reply:
https://github.com/apache/incubator-nuttx/pull/181